doc/source/conf.py

```diff
@@ -252,23 +252,36 @@
 
 nitpicky = True
 nitpick_ignore = [
-    ("py:obj", "color_dict"),
     ("py:class", "skimage.transform._geometric._GeometricTransform"),
-    ("py:class", "skimage.transform._geometric.ProjectiveTransform"),
-    ("py:class", "skimage.feature.util.DescriptorExtractor"),
-    ("py:class", "skimage.feature.util.FeatureDetector"),
-    ("py:class", "skimage.measure.fit.BaseModel"),
-    ("py:exc", "NetworkXError"),
-    ("py:obj", "Graph"),
-    ("py:obj", "Graph.__iter__"),
-    ("py:obj", "__len__"),
-    ("py:class", "_GeometricTransform"),
-    ("py:obj", "convert"),
-    ("py:obj", "available_plugins"),
-    ("py:obj", "skimage.io.collection"),
-    ("py:obj", "skimage.io.manage_plugins"),
-    ("py:obj", "skimage.io.sift"),
-    ("py:obj", "skimage.io.util"),
+    # ^ skimage.transform._geometric.{FundamentalMatrixTransform,PiecewiseAffineTransform,PolynomialTransform,ProjectiveTransform}
+    (
+        "py:class",
+        "skimage.feature.util.DescriptorExtractor",
+    ),  # skimage.feature.{censure.CENSURE/orb.ORB/sift.SIFT}
+    (
+        "py:class",
+        "skimage.feature.util.FeatureDetector",
+    ),  # skimage.feature.{censure.CENSURE/orb.ORB/sift.SIFT}
+    (
+        "py:class",
+        "skimage.measure.fit.BaseModel",
+    ),  # skimage.measure.fit.{CircleModel/EllipseModel/LineModelND}
+    ("py:exc", "NetworkXError"),  # networkx.classes.graph.Graph.nbunch_iter
+    ("py:obj", "Graph"),  # networkx.classes.graph.Graph.to_undirected
+    ("py:obj", "Graph.__iter__"),  # networkx.classes.graph.Graph.nbunch_iter
+    ("py:obj", "__len__"),  # networkx.classes.graph.Graph.{number_of_nodes/order}
+    (
+        "py:class",
+        "_GeometricTransform",
+    ),  # skimage.transform._geometric.estimate_transform
+    ("py:obj", "convert"),  # skimage.graph._rag.RAG.__init__
+    ("py:obj", "skimage.io.collection"),  # (generated) doc/source/api/skimage.io.rst
```

Comment:
It should be possible to fix these last four, but I am also out of ideas here. From what I understood, `skimage.io` is doing some funky doc injection, which may be the cause of the errors. In the generated `.rst` file, exactly these four sections are misaligned:
<img width="321" alt="image" src="https://github.com/scikit-image/scikit-image/assets/1315589/50100dcf-1745-4153-95b5-8a5f9794ccf8">

Reply:
Okay, I have found and fixed the alignment issue, but that is not enough 😄.